webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -276,21 +281,26 @@ private static PdfReader createPdf(String globalTitle, Date creationDate, Scramb
                 attemptRequest.event = "333bf";
                 attemptRequest.colorScheme = scrambleRequest.colorScheme;
 
-                PdfReader pdfReader = createPdf(globalTitle, creationDate, attemptRequest, locale);
+                PdfReader pdfReader = new PdfReader(createPdf(globalTitle, creationDate, attemptRequest, locale, null).toByteArray());
```

Comment:
Could you put a quick comment here about why password is null?

Reply:
Done!